monkey/infection_monkey/exploit/drupal.py

```diff
@@ -0,0 +1,175 @@
+"""
+Remote Code Execution on Drupal server - CVE-2019-6340
+Implementation is based on:
+    https://gist.github.com/leonjza/d0ab053be9b06fa020b66f00358e3d88/f9f6a5bb6605745e292bee3a4079f261d891738a.
+"""
+
+import logging
+import re
+import requests
+from urllib.parse import urljoin, urlparse
+from infection_monkey.exploit.web_rce import WebRCE
+
+__author__ = 'Ophir Harpaz'
+
+LOG = logging.getLogger(__name__)
+
+
+def remove_port(url):
+    parsed = urlparse(url)
+    with_port = f'{parsed.scheme}://{parsed.netloc}'
+    without_port = re.sub(':[0-9]+$', '', with_port)
+    return without_port
```

Comment:
This should be split into `remove_port_from_url` and `remove_path_from_url`. Both of these should end up in `monkey/common/network/network_utils.py` and the file itself should be renamed from `network_utils` to `url_parsing_utils`.

Reply:
1. There is no `network_utils` file the path you specified. There is one under `monkey_island` but it's definitely not just URL-parsing functions, so I'm not sure what's best to do.
2. I'm not sure why you suggest to split this function into two.

Reply:
Fixed (apart from splitting the function) 
1ae8ecff623e03baafca3727482bf6d05b17bb52